packages/js/admin-e2e-tests/src/specs/homescreen/welcome-modal.ts

```diff
@@ -0,0 +1,39 @@
+/**
+ * Internal dependencies
+ */
+import { Login } from '../../pages/Login';
+import { OnboardingWizard } from '../../pages/OnboardingWizard';
+import { WcHomescreen } from '../../pages/WcHomescreen';
+import { resetWooCommerceState } from '../../fixtures';
+
+/* eslint-disable @typescript-eslint/no-var-requires */
+const { afterAll, beforeAll, describe, it } = require( '@jest/globals' );
+/* eslint-enable @typescript-eslint/no-var-requires */
+
+const testAdminHomescreenWelcomeModal = () => {
+	describe( 'Homescreen welcome modal', () => {
+		const profileWizard = new OnboardingWizard( page );
+		const homeScreen = new WcHomescreen( page );
+		const login = new Login( page );
+
+		beforeAll( async () => {
+			await login.login();
+			await resetWooCommerceState();
+			await profileWizard.navigate();
+			await profileWizard.skipStoreSetup();
+		} );
+
+		afterAll( async () => {
+			await login.logout();
+		} );
+
+		it( 'should not show welcome modal', async () => {
```

Comment:
I am not entire opposed to having a whole separate test condition for the welcome modal, but I wonder if something like this should be part of the `complete-onboarding-wizard` spec, given it is specific to onboarding?

If not, I would suggest changing the naming of this test to be more generic and Homescreen focused. `describe('Homescreen` (this would also mean updating the filename.
That way we could add more home screen specific specs in here, like checking if `notes` are rendered for example and if the Stats overview is rendered.

Reply:
Good idea @louwie17! I moved the test to the file `complete-onboarding-wizard` and renamed the test description from "Homescreen welcome modal" to just "Homescreen" in the commit 35a2eb1